sdf.md

```diff
@@ -1293,7 +1294,7 @@ All these definitions share some common qualities as discussed in {{common-quali
 ## sdfObject
 
 The sdfObject keyword denotes a group of zero or more Object definitions.
-Object definitions may contain or include definitions of Properties, Actions, Events declared for the object, as well as data types (sdfData group) to be used in this or other Objects.
+Object definitions may contain or include definitions of Properties, Actions, Events declared for the Object, as well as data types (sdfData group) to be used in this or other Objects.
```

Comment:
This part seems a bit redundant, but I guess removing it would make the sentence too imprecise?

```suggestion
Object definitions may contain or include definitions of Properties, Actions, Events, as well as data types (sdfData group) to be used in this or other Objects.
```

Reply:
That change would miss that PAE are "declared" (i.e., declarations).